.changeset/pre.json

```diff
@@ -0,0 +1,14 @@
+{
+  "mode": "pre",
+  "tag": "alpha",
+  "initialVersions": {
+    "@apollo/composition": "2.3.3",
```

Comment:
No change needed, but I just realized this gets kinda wonky if we land a patch to `main`?

Reply:
I don't think so, because `next` will just continue to be on `2.4.0-alpha.*` until we exit `pre` mode, right? I don't think that releasing patches out of main should matter to it.

Reply:
Right, for some reason I was thinking it'd be `2.3.4` on next but that's definitely not the case.